Formula/libtiff.rb

```diff
@@ -19,65 +18,9 @@ class Libtiff < Formula
   depends_on "xz" => :optional
 
   # Patches are taken from latest Fedora package, which is currently
```

Comment:
This text should be removed since the patches are removed but that can be done when merging, no need to push a new commit if its just for that.